internal/manifests/manifests.go

```diff
@@ -172,8 +172,6 @@ func (m *Manifests) ListClusterManifestsInternal(ctx context.Context, params ope
 		}
 		if isUserManifest {
 			manifests = append(manifests, &models.Manifest{FileName: filename, Folder: folder})
-		} else {
```

Comment:
regarding line 154 - if the cluster doesn't exist please replace the "Object not found" message and use something like https://github.com/openshift/assisted-service/blob/master/internal/bminventory/inventory.go#L1535